libs/utils/env.py

```diff
@@ -439,6 +441,54 @@ def _init_target(self, force = False):
             self.__modules = ['bl', 'cpufreq']
             platform = Platform(model='pixel')
 
+        # Initialize gem5 platform
+        elif self.conf['board'].upper() == 'GEM5':
+            self.nrg_model = None
+            self.__modules=['cpufreq']
+
+            system_info = self.conf['board_settings']['system']
+            simulator_info = self.conf['board_settings']['simulator']
+            simulator_args = ""
+            platform_args = ""
+
+            # Get gem5 binary arguments
+            for args in simulator_info['args']:
+                simulator_args += args + " "
+            # Required for telnet terminal
+            simulator_args += "--listener-mode=on"
+
+            # Get platform arguments
+            for args in system_info['platform']['args']:
+                platform_args += args + " "
+            # Mandatory arguments
+            platform_args += " --kernel "     + system_info['kernel'] + \
+                             " --dtb "        + system_info['dtb'] + \
+                             " --disk-image " + system_info['disk']
+
+            # Gather all arguments
+            args = simulator_args + " " + \
+                   system_info['platform']['description'] + " " + \
+                   platform_args
+
+            # Setup virtio
+            virtio_args = "--workload-automation-vio={}"
```

Comment:
Is this missing a call to `str.format` or is this the legit arg? If the latter please add a comment.